blockstore/apps/bundles/tests/factories.py

```diff
@@ -1,24 +1,24 @@
 """ Factories for bundle models. """
-
-import factory
+# Not in the main package list, and not named the same as the requirement besides.
+import factory  # pylint: disable=import-error
```

Comment:
I think that the error here is due to:

 https://github.com/FactoryBoy/factory_boy/issues/780